src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -34,7 +34,7 @@ import type { BlastJobResultResponse } from 'src/content/app/tools/blast/types/b
 import type { SubmittedSequence } from 'src/content/app/tools/blast/types/blastSequence';
 
 export type BlastSubmissionPayload = {
-  species: Species[];
+  species: Partial<Species>[];
```

Comment:
This looks suspiciously permissive.

Reply:
It is a test, so does it really matter?

Reply:
What do you mean a test? This is not a test file.

Reply:
What I mean is this is `createBlastSubmissionPayload` is used in blastEpics.test

Reply:
Happy to update it, but the question is, is it necessary.

Reply:
Sorry my bad. It is not just used on test. Will push my changes once you resolve the conflict